test/WebJobs.Script.Tests/ScriptStartupTypeDiscovererTests.cs

```diff
@@ -9,8 +9,10 @@
 using Microsoft.Azure.WebJobs.Extensions;
 using Microsoft.Azure.WebJobs.Extensions.Http;
 using Microsoft.Azure.WebJobs.Extensions.Storage;
+using Microsoft.Azure.WebJobs.Script.BindingExtensionBundle;
```

Comment:
Nit: Can you please change the name space? ExtensionBundle  sounds more appropriate than BindingExtensionBundle